File: Exercicios_guanaraba/Com_Ruby/exercicios/ex029_Radar_eletronico.rb
=begin

Escreva um programa que leia a velocidade de um carro.
Se ele ultrapassar 80km/h, mostre uma mensagem deizendo que ele doi multado.
A multa vai custar R$7,00 por cada Km acima do limite.

=end



puts 'Insira a velocidade atingida: '

velocidade = gets.chomp.to_i

multa = (velocidade - 80) * 7

if velocidade > 80
    puts "Você foi multado em R$#{multa}"
else
    puts 'VocÊ esta no limite permitido'
end

File: Exercicios_guanaraba/Python3/Exercicios/Aula 10 Condições pt1/ex029 Radar eletronico.py
'''
Escreva um programa que leia a velocidade de um carro.

Se ele ultrapassar 80km/h, mostre uma mensagem dizendo que ele foi multado.

A multa vai custar R$7,00 por cada KM acima do limite.

'''

km=int(input('Insira a velocidade atingida: '))

if km > 80:
    print('Voce foi multado em R${}'.format(7*(km-80)))
else:
    print('Você esta respeitando o limite, parabéns')


